### DISCUSSION TITLE
What do you think about my Karlskrona theme?

### DISCUSSION THREAD

**Discussion by pftnhr** [datenstrom/community#901] [Ask a question] [answered]
Since almost every time I install Yellow for a new project I go through the code to delete the CSS classes and add semantic elements (header, main, footer, section, article, etc.) in preparation for using Pico, I thought I'd just create a theme to get the next project going faster.

I only implemented the pico.css in the theme. For my projects, I also use the sass files.

You can find the theme here: [yellow-karlskrona](https://github.com/pftnhr/yellow-karlskrona)

**Comment by schulle4u** [comment, marked as answer]
Finally another developer who shares my bent for semantic elements. 👍 I see you have added a class attribute to the opening `<body>` element. Does this also work with the edit extension? In some situations the edit extension caused loading issues with my wittstock theme, thats why I left a div container with the necessary  classes in the layout. I also had problems using the immersive readers in different browsers which I still couldn't solve.

**Reply by pftnhr** [reply]
Thank you very much Steffen! I think semantic elements make a lot of things easier.

The edit extension works, but is not styled. Maybe I'll do that later.

**Reply by schulle4u** [reply]
Just gave it a try, and unfortunately your theme has the same small issue as mine. You can test it by visiting the edit location of your website while logged out. Then enter some wrong credencials to make Yellow display a "try again" link. This link won't work if your theme is loaded. From what i found out the edit extension expects the presence of a top-level div element with appropriate page layout classes after the opening body tag.

**Reply by pftnhr** [reply]
Thanks a lot for testing! I'll take a look at it later and fix it.

**Reply by pftnhr** [reply]
I have removed the classes from the body tag and it still doesn't work. I then set up a fresh Yellow installation and tested the procedure there too, with the result that it doesn't work there either.

It should call `#data-action-login` but there is nothing. I will write a bug report.
